test/TEST-70-DELEGATE-THREADED/test.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -e
+
+TEST_DESCRIPTION="Test threaded cgroup in unit with Delegate=yes"
```

Comment:
Can you fold this into the existing TEST-19-DELEGATE? Each new test has additional setup/teardown costs

Reply:
I wasn't aware of the costs. Good idea! (I've moved the test in new version.)